code/game/objects/items/dice.dm

```diff
@@ -222,15 +225,19 @@
 
 	var/fake_result = roll(sides)//Daredevil isn't as good as he used to be
 	var/comment = ""
-	if(sides == 20 && result == 20)
-		comment = "NAT 20!"
-	else if(sides == 20 && result == 1)
-		comment = "Ouch, bad luck."
+	if(sides > 1)  // less comment spam
+		if(result == sides)
+			comment = "NAT [sides]!"
+		if(result == 1)
+			comment = "Ouch, bad luck."
 	update_appearance()
-	if(initial(icon_state) == "d00")
-		result = (result - 1)*10
+	result = manip_result(result)
 	if(special_faces.len == sides)
+		comment = ""  // its not a number
 		result = special_faces[result]
+		if(!ISINTEGER(x))
```

Comment:
you dont seem to have an x anywhere in the proc

Reply:
can you put in a "suggested changes" to change that to ISINTEGER(result)? im not able to edit it at the moment